src/rt-app_parse_config.c

```diff
@@ -811,7 +811,7 @@ parse_thread_data(char *name, struct json_object *obj, int index,
 		data->phases = malloc(sizeof(phase_data_t) * data->nphases);
 		parse_thread_phase_data(obj,  &data->phases[0], opts, (long)data);
 		/* Get loop number */
-		data->loop = -1;
+		data->loop = get_int_value_from(obj, "loop", TRUE, -1);
 	}
```

Comment:
But with this set we actually discard phase->loop and not thread->loop when there is only a single phase.
So with the below
```
{
        "tasks" : {
                "thread0" : {
                        "instance" : 1,
                        "cpus" : [0],
                        "loop" : 3,
                        "phases" : {
                                "phase1" : {
                                        "loop" : 4,
                                        "run" :  1000000
                                }
                        }
                }
        },
        "global" : {
                "duration" : -1,
                "calibration" : 585,
                "default_policy" : "SCHED_OTHER",
                "pi_enabled" : false,
                "lock_pages" : false,
                "logdir" : "./",
                "log_basename" : "rt-app"
        }
}
```
We will be looping for 3 times, instead of 12, which is a sane thing to do?